internal/rollup/rollup_bundle.bzl

```diff
@@ -537,7 +537,11 @@ def _rollup_bundle(ctx):
         )
 
     return [
-        DefaultInfo(files = depset(files), runfiles = ctx.runfiles(files)),
+        DefaultInfo(
+            files = depset(files),
+            # NB: we don't include any runfiles here since they would always be built
```

Comment:
IDK if this comment is really required. Almost seems obvious that es5min is not a "runtime depdency/requirement" of the default es5min files 🤷‍♂

Reply:
I mostly added this to remind us not to re-introduce the problem, since I didn't add a test that asserts that default actions don't run

(the only test I could think of is similar to my repro where the default file fails to build, but that would be hard to build into the rollup_bundle rule)
(maybe if we could assert on the action graph of a build, that would be the right way to assert what we want here)